packages/ckeditor5-dev-utils/tests/translations/translationservice.js

```diff
@@ -17,7 +17,7 @@ const mockery = require( 'mockery' );
 describe( 'translations', () => {
 	describe( 'TranslationService', () => {
 		const sandbox = sinon.sandbox.create();
-		let translationService, stubs;
+		let translationService, TranslationService, stubs;
```

Comment:
> Most probably it will be repeated in each test scenario but the whole file will be easier to read.

Yes, I wanted to avoid it, but your idea is probably better.